base_layer/wallet/migrations/2024-12-05-110700-payment_id/up.sql

```diff
@@ -0,0 +1,6 @@
+
+ALTER TABLE inbound_transactions
+    ADD payment_id BLOB NULL;
+
+ALTER TABLE outbound_transactions
+    ADD payment_id BLOB NULL;
```

Comment:
cant see why this is required?
It should be on the outputs already

Reply:
To conevert between the different types transactions - inbound, outbound and completed.